### ISSUE TITLE
DefaultTexts.SEND_CODE does not match translation key in ui-angular package

### ISSUE THREAD

**Issue by coyoteecd** [aws-amplify/amplify-ui#2653]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have tried disabling all browser extensions or using a different browser
- [X] I have tried deleting the node_modules folder and reinstalling my dependencies
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

Angular

### Which UI component?

Authenticator

### How is your app built?

Angular CLI

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### Please describe your bug.

We are trying to use the translation keys in DefaultTexts when providing custom translation overrides for aws-amplify and found out that SEND_CODE key is differently cased between the react and angular implementation of the Reset Password component.

React uses "Send code":
https://github.com/aws-amplify/amplify-ui/blob/bc882121a8d0e005099e2827c3bde68d56647100/packages/react/src/components/Authenticator/ResetPassword/ResetPassword.tsx#L54

Angular uses "Send Code":
https://github.com/aws-amplify/amplify-ui/blob/35c8b11fab6703f3ca24fff1edd3660990674892/packages/angular/projects/ui-angular/src/lib/components/authenticator/components/reset-password/reset-password.component.ts#L18

### What's the expected behaviour?

I expect the DefaultTexts to contain keys that work correctly across all components (to avoid hard-coding translations, which is not very friendly when upgrading).

### Help us reproduce the bug!

Putting the line below in the ngOnInit of a project with the authenticator component doesn't work (shows Send Code):
```ts
I18n.putVocabulariesForLanguage('ja', {
  [DefaultTexts.SEND_CODE]: 'Some value'
});
```

This does work:
```ts
I18n.putVocabulariesForLanguage('ja', {
  'Send Code': 'Some value'
});
```

### Code Snippet

```typescript
// Put your code below this line.

```

**Comment by wlee221** [comment]
Hi @coyoteecd! You're right, there are inconsistency with translations. We'll add this onto the next major release to not break developers using "Send Code" to translate. I'll keep this thread updated.

FYI, you can see the progress on [`next-release/main`](https://github.com/aws-amplify/amplify-ui/tree/next-release/main) branch.

**Comment by wlee221** [comment]
This should be resolved now (see #2735), closing. Let us know if this still persists, thanks!
